workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboardPage/PurchasedDashboardPageUtil.ts

```diff
@@ -56,7 +56,7 @@ export const customerPermissionDescriptions : PermissionDescription[] = [
   {
     permissionName: 'Purchase apps and solutions',
     permissionTooltip: 'Purchase new apps and versions',
-    permitedRoles: ['Account Administrator', 'Account Buyer'],
+    permitedRoles: ['Account Buyer'],
```

Comment:
Please, change it here and other parts of the code.
```suggestion
    permittedRoles: ['Account Buyer'],
```